core/src/main/java/bisq/core/btc/nodes/BtcNodes.java

```diff
@@ -70,6 +70,7 @@
                         // Devin Bileck
                         new BtcNode("btc1.bisq.services", "lva54pnbq2nsmjyr.onion", "172.105.21.216", BtcNode.DEFAULT_PORT, "@devinbileck"),
                         new BtcNode("btc2.bisq.services", "qxjrxmhyqp5vy5hj.onion", "173.255.240.205", BtcNode.DEFAULT_PORT, "@devinbileck"),
+                        new BtcNode(null, "wubwzaadboxwiffa.onion", null, BtcNode.DEFAULT_PORT, "@devinbileck"),
```

Comment:
I notice the hostname and ip-address arguments are null.   Does that matter?

Reply:
The clearnet host/address is only used for people running regtest or if someone is intentionally running on clearnet only.
This node does not have a static IP so I omitted them.

Reply:
My mistake.  I should have figured that out.